k6/src/k6/utils.ts

```
@@ -0,0 +1,93 @@
+    public static buildMessagePayload(): ArrayBuffer {
+        const messagePayload = `GET /?startTime=${Date.now()} HTTP/1.1\r\nHost: ${getDestination()}\r\n\r\n`;
+        //console.log("Message sent payload:" + JSON.stringify(messagePayload)); 
+        return Utils.stringToArrayBuffer(messagePayload)
+    }
+
+    public static unpackMessagePayload(messagePayload: ArrayBuffer): string {
+        let httpResponse = Utils.arrayBufferToString(messagePayload);
+        //console.log("Message received payload:" + JSON.stringify(httpResponse));
+        switch (getDestination()) {
+            case destinations.ECHO_SERVICE:
+                const body = httpResponse.substring(httpResponse.indexOf("{\"message\""), httpResponse.length);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)